runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -63,4 +64,48 @@ default String filteredRegex()
 	{
 		return "";
 	}
+
+	@ConfigItem(
+		keyName = "addDefaultFilterRegex",
+		name = "Add default filter regex",
+		description = "Add the default chat filter in addition to your own filtered words and regexes",
+		position = 4
+	)
+	default boolean addDefaultFilterRegex()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "whitelistYourself",
+		name = "Whitelist Yourself",
```

Comment:
Changing these options to `Filter x` and default true would be more clear to a user.

Reply:
done